spec/support/test/stripe_mock_helper_spec.rb

```diff
@@ -18,8 +18,10 @@
   describe "#start" do
     it 'is safely reentrant' do
       StripeMockHelper.mock do
-        # create a plan 
-        StripeMockHelper.stripe_helper.create_plan(id: 'test_str_plan', amount:0, currency: 'usd', interval: 'year', name: 'test PLan')
+        # products are now required in plans as of Stripe gem 5.0
+        product = StripeMockHelper.stripe_helper.create_product
+        # create a plan
+        StripeMockHelper.stripe_helper.create_plan(id: 'test_str_plan', amount:0, currency: 'usd', interval: 'year', name: 'test PLan', product: product.id)
```

Comment:
nit: spacing `amount:0`

Reply:
Question: Does this require us to make changes to BillingPlan or other places and now require a "plan" as well? All of these changes are in factory and test code and no implementation code.